plugins/other/sshd_log

```diff
@@ -28,18 +29,35 @@
 #env.logfile /var/log/messages
 #env.category users
 #
+# config example with journald
+#[sshd_log]
+#group systemd-journal
+#env.logfile journald
+#env.journald_unit sshd.service
+#
 
 LOG=${logfile:-/var/log/secure}
 CATEGORY=${category:-system}
+JOURNALD_UNIT=${journald_unit:-sshd.service}
 
 
 if [ "$1" = "autoconf" ]; then
-        if [ -r "$LOG" ]; then
-                echo yes
-                exit 0
+        if [ $LOG == "journald" ]; then
```

Comment:
the test "==" is a bashishm - please replace it with "="
`shellcheck -s dash sshd_log` will give you even more useful hints (e.g. regarding quoting)